continuous_integration/test_script.cmd

```diff
@@ -1,5 +1,6 @@
-set DEFAULT_PYTEST_ARGS=-vlrx --timeout=60
+set DEFAULT_PYTEST_ARGS=-vlrx --timeout=60 --cov=benchopt
 
 call activate %VIRTUALENV%
 
 pytest --junitxml=%JUNITXML% %DEFAULT_PYTEST_ARGS%
+codecov
```

Comment:
This script is not run as it is for windows CI.

Also, in pipeline, I'd rather add a separate step in the pipeline that only run if the test step was successful.